docs/src/examples/components/Dialog/Content/DialogExampleContent.shorthand.tsx

```diff
@@ -7,6 +7,7 @@ const DialogExampleContent: React.FC = () => (
     confirmButton="Confirm"
     content="Are you sure you want to confirm this action?"
     header="Action confirmation"
+    headerAction={{ icon: 'more' }}
```

Comment:
plase add `title` prop to this and the next example - let's educate the devs that they need to provide the text for icon only buttons

Reply:
added